Modules/@babylonjs/react-native/shared/XrAnchorHelper.h

```diff
@@ -183,3 +183,47 @@ namespace Babylon::Plugins::NativeXr
 }
 #endif
 #endif
+
+#if __has_include("IXrContextARKit.h")
+#include "IXrContextARKit.h"
+#if __has_include("jsi/jsi.h")

```

Comment:
This feels like something that should be handled at the CMake level (like we do with AppRuntime* etc) instead of doing compile-time if's

Reply:
these files can be referenced by anyone independent of whether they use cmake